shell/models/__tests__/management.cattle.io.node.ts

```diff
@@ -0,0 +1,82 @@
+import MgmtNode from '@shell/models/management.cattle.io.node';
+
+describe('class MgmtNode', () => {
+  const t = jest.fn(() => 'baz');
+  const ctx = { rootGetters: { 'i18n/t': t } };
+
+  const resetMocks = () => {
+    // Clear all mock function calls:
+    jest.clearAllMocks();
+  };
+
+  it('should not return addresses if they are not present in the resource status, the internalNodeStatus, or the rkeNode key in status', () => {
+    const mgmtNode = new MgmtNode({ status: {} }, ctx);
+
+    expect(mgmtNode.addresses).toStrictEqual([]);
+    resetMocks();
+  });
+  it('should return addresses if they are present directly on the resource status', () => {
+    const mgmtNode = new MgmtNode({ status: { addresses: ['foo'] } }, ctx);
+
+    expect(mgmtNode.addresses).toStrictEqual(['foo']);
+    resetMocks();
+  });
+  it('should return addresses if they are not present directly on the resource status but are on "status.internalNodeStatus"', () => {
+    const mgmtNode = new MgmtNode({ status: { internalNodeStatus: { addresses: ['bar'] } } }, ctx);
+
+    expect(mgmtNode.addresses).toStrictEqual(['bar']);
+    resetMocks();
+  });
+  it('should return an internalIp if addresses includes an object with an appropriate type and address', () => {
```

Comment:
You have 4 tests where you repeat `should return an internalIp if`. It should be contained in the same description having this part of text and remove it from the test. It will helps it also to be conciser.

Reply:
Done